bridge/client/app/_components/_abstract/ktb-overlay.component.ts

```diff
@@ -0,0 +1,33 @@
+import { ElementRef, OnDestroy, OnInit } from '@angular/core';
+import { OverlayService } from '../../_directives/overlay-service/overlay.service';
+import { Subject } from 'rxjs';
+import { OverlayRef } from '@angular/cdk/overlay';
+
+export abstract class KtbOverlayComponent implements OnInit, OnDestroy {
```

Comment:
@Component decorator is missing here, but required by the angular features (dependency injection, etc.)